pillar/kube-minion.sls

```diff
@@ -1,2 +1,4 @@
 mine_functions:
   network.ip_addrs: [eth0]
+
```

Comment:
It would be nice to have a comment here explaining what `pod_infra_container_image` is. :-)